### DISCUSSION TITLE
关于某位协作者

### DISCUSSION THREAD

**Discussion by allMagicNB** [Meloong-Git/PCL#5136] [讨论]
> 那么如果有区别的话。
> 
> - #2863
> - #3039
> - #4336
> 
> 这些又是怎么重的？

我敢说，要是这次另一个协作者没多想，那么这个 Issue 也会很快被 Close。
一个很碰巧的事：这几个 Issue 都是由同一个人查出来的重复，这人现在还是一名协作者，现在还在处处挑我毛病。
Collaborator 查重查错是能理解的事，毕竟人人都会有考虑不周的时候。
那么连着三次，同一个问题，同一个说辞，该如何解释？这显然是说不通的，这只能说明这位协作者不认真。
可能又有人会说：“你有什么身份在这说别人？”我只能说，你爱听不听，不听也罢。
- #4650 某次他查重查错过，我暗示过，可他直到我把原 Issue 贴出来他才知道重了。虽然当时是玩笑的语气，但是说的事却是真的。没记错的话当时查重很容易就查到了。

- #4979 他直接带“等待确认” Label 提交 Issue，龙猫甚至直接贴“处理中” Label 了，但它确实是重了，最后这 Issue 也被 Close 了。

- #5045 他在拿社区群随便讨论出来的答案误导人。这个答案甚至只需要在源代码中随便搜搜就能找到。此 Issue 的作者在乱说，他一个 Collaborator 也在乱说。我说话冲了点，但是实话，他于是又过来指责我语气，甚至于有指责我的时间都不愿意把他的 Comment 改一改，甚至于到现在都还没改。

在我被龙猫取消 Collaborator 后，他马上就用社区组织 Owner 权限把我降级为 Member。在我发现后问他时，他选择冷处理，最后还是我找组织创建人设置回来的。这一点 @WForst-Breeze 能证实。

我这人有时候说话语气确实比较冲，也因为对部分功能了解不深带来误会。志愿者没有客服的工资，自然也就不一定有客服的态度。不是所有人都能憋住生气。

**Comment by Chiloven945** [comment]
所以说您发该 Discussion 的目的与意义是为了什么？是为了抨击您文中所提到的 Collaborator 还是纯粹发泄自己的恨意？

如果您在论证一个人是否能够胜任 Collaborator 这个职位，那么您**不应该通过论证他在处理事件时犯的常识性错误以此来证明**。每个人在处理问题时都会出现常识性错误，提出错误的解决办法或建议等，这是可被饶恕的。错误判断不等同于“不认真”，更可能是因为工作量大、细节疏忽所致。并不否认当事人在处理问题时可能犯的错误，但多次出现这种问题真正所需要的是正确的沟通方式，而不是像您这样写一篇“抨击文章”去全盘否定去证实别人不胜任这个职位。

正确论述一个人是否能胜任这个职位的方法，**应该是判断他处理这件事情所运用的语气与态度**。因为将个人不满与具体问题混为一谈并不利于解决问题，协作者的态度问题等与讨论 Issue 是否重复等事件也并无直接关联。社区讨论和合作中，语气与沟通的方式至关重要。志愿者确实没有客服的工资，因为没有人专门来发工资，参与社区协作本身就意味着需要承担一定的责任与义务。如果无法承担这份责任与义务，那显然是不适合的。语气“冲”所影响的是讨论效率，并且可能导致本身所存在的误解扩大化，同时也会导致别人很难接受本就正确的建议，这带来的影响是要严重得多的。

**Reply by allMagicNB** [reply]
你但凡点进过我引用的 Issue 你就知道我在说谁。

**Reply by Chiloven945** [reply]
> 你但凡点进过我引用的 Issue 你就知道我在说谁。

您但凡把当事人 @ 出来也不会使人混淆视听。

**Reply by allMagicNB** [reply]
> > 你但凡点进过我引用的 Issue 你就知道我在说谁。
> 
> 您但凡把当事人 @ 出来也不会使人混淆视听。

您但凡多看几眼都不会这么认为。

**Reply by allMagicNB** [reply]
那么，不论语气论态度如何？
在我发现重复后，做出一定提示，他在做什么？他依旧认定没有重复。
在我问他 Owner 权限什么时候没的时，他又在做什么？在群里闲的聊天都不愿意回我，最后查出的记录就是他把我降级了。我发出了记录截屏，他还是选择冷处理，即使我贴出证据，他也还在冷处理。

**Comment by LTCatt** [comment]
如果我没有理解错的话，你是在说 @JingHai-Lingyun 的问题，而非 @3gf8jv4dv ？

---

如果你觉得 @JingHai-Lingyun 的做法存在不妥，想要寻求解决，正确的做法应该是： **直接艾特他，然后把你上面指出的 issue 的地址贴出来。** 之后，大家才能讨论具体存在的问题。
然而你却反而艾特了一位不相干的 Collaborator，然后对另外一位 Collaborator 进行批评，还刻意地没有提当事人的名字。
我确实不知道你这样做是为了什么，但结果上，这种做法 **不仅解决不了问题，还只会加剧社区中的对立。**

---


如果你确实想指出问题， **请正大光明地写出来** ，并且把对应的 issue 贴出来。
现在这种语焉不详地指责他人行为，和在别人背后说风凉话无异。
总之，我非常不赞同这样的行为。

**Comment by allMagicNB** [comment]
原本是想将这段话带到 Comment 中的，想了想还是决定单独开 Discussion。然后复制粘贴忘删 @ 了。
本文没有任何地方在指责 @3gf8jv4dv 哪里做的不对，相反，还得感谢他的提醒。如果你从哪读出了我对他不满，那么只能说明你想太多了。

**Comment by WorldHim** [comment]
> 在我被龙猫取消 Collaborator 后，他马上就用社区组织 Owner 权限把我降级为 Member。在我发现后问他时，他选择冷处理，最后还是我找组织创建人设置回来的。

这是我最不理解的地方。
首先这与主题无关
其次我并不认为你们互相争 Owner 有什么意义。这只是一个社区组织，Owner 与否并不能说明什么，你需要这个权限来做什么呢？我对你们双方的做法不予置评，但有这时间还不如反思自己

**Reply by allMagicNB** [reply]
此 Discussion 并非针对于 Collaborator 身份，而是针对个人。
其次，Owner 能够对新成员发送邀请，删除仓库，总之管理组织时需要。

**Comment by wuliaodexiaoluo** [comment]
>他直接带“等待确认” Label 提交 Issue

对于这个问题，我的建议是最好能像萌娘百科的行政方针那样结合社区现有的讨论出一份类似的规范，明确什么情况下建议做什么，什么情况下不建议做什么
也可以链接一些比较有用的东西，这些有用的东西(#1592 and more)已经彻底埋没在历史了
这甚至可以解释一下 Label 应该怎么用，什么时候用，谁能用（（（

可以充分利用下 GitHub Wiki (((

~当然最好加个**不要墨守成规**（（（~

**Reply by allMagicNB** [reply]
除非查重非常彻底，否则一般不建议在提交 Issue 时直接更换模板 Label。

**Comment by Deep-Dark-Forest** [comment]
我不知道你是怎么做到在 QQ 群里和 GitHub 里表现出完全两个人的

> Collaborator 查重查错是能理解的事，毕竟人人都会有考虑不周的时候。

![Image_293717312922890_edit_293894189448904.png](https://github.com/user-attachments/assets/a36315ef-bfce-4fd7-947c-239b509e11b4)

**《能理解》指你直接对没有 “仔细查重” 的 collaborator 吐脏字**

（他说这句话是因为 #4777 我没有采取他认为的 https://github.com/Hex-Dragon/PCL2/issues/4777#issuecomment-2365378030 就认为我是错的，然后最终不还是他成了小丑）

---

实际上如果你要挑的话几乎每个 Collaborator 都做过不对的事。如果你就是在挑凌云的毛病，其实从某种程度上也能理解吧，因为当时就是因为他的举报你才丢掉了 Collaborator 对他怀恨在心想找机会报复不是吗？

**Reply by allMagicNB** [reply]
我也觉得你在 GitHub 上和在 QQ 上也像两个人。
#4777，从日志来看，确实是 BMCLAPI 返回 403 无法下载，龙猫也表示不完全认同，那么谁是小丑？

**Reply by allMagicNB** [reply]
我觉得你有严重的妄想症。

**Reply by Deep-Dark-Forest** [reply]
> 我觉得你有严重的妄想症。

确实，你的关注点全都在你能找到理的事情上

**Comment by LTCatt** [comment]
我的意见是，我不认为社区中某个人偶尔查重查漏了是应该被指责的事。
现在几千个 issue，我自己都经常忘记有的问题我已经看过了。例如 #5134（#2467）的技术实现，我其实在 #2689 就调查过并且确认不可行了，但一年多过去完全忘了，还重新调查了一遍。
开发者都尚且如此，要求社区查重得要查得没问题，未免过于苛责了。
如果这个人查漏了，另一个人也可能发现，然后改成重复，这完全没有什么影响，不是吗？
说到底，大家本就是自愿来帮帮忙的，我对此已经很感激了，如果因此还要背负压力，或者“整错了就要被开批斗会”，这就有点令人伤心了。

**Reply by wuliaodexiaoluo** [reply]
>>直接带“等待确认” Label 提交 Issue
>
>我不认为社区中某个人偶尔查重查漏了是应该被指责的事。

但这种直接带等待确认的情况我认为有必要减少
如果带的是新反馈，肯定会有第二个乃至第三个去查有没有重复
但这种一般不会有人去看，除非查的人也是协助者或者特别认真的人（（（
只能说这个标签的可信度有点过高了（（（

**Reply by allMagicNB** [reply]
我对此是认同的。
但他的行为我觉得不能称上“偶尔”，同一个问题，他连续三次都能查错。

**Reply by LTCatt** [reply]
hmm…… #2863、#3039、#4336 要说 “同一个问题，他连续三次都能查错”，说法上可能有点过火了，这几个 issue 和 #2467 是蛮相似的。不过确实 3gf8jv4dv 的处理方式更好，可以在转等待确认之前更加谨慎一些。
问题其实是 #4979，他自己作为提交者交了重复 issue，并且直接以等待确认的 tag 交上来的，无论如何不应该直接跳过挂新反馈让社区进行二次确认的流程，这点确实需要注意 @JingHai-Lingyun

**Reply by allMagicNB** [reply]
#2467 仅为做两个按钮方便用户手动标记客户端/服务端。

**Reply by iPodtouch6557** [reply]
> 以下只是一些个人猜想，仅供参考（可以简单理解为纯瞎猜的）：

删了吧，影响不好，就现在的情况看（……）

**Comment by SALTWOOD** [comment]
实际上协管没那么重要……积极参与社区就很不错了……遇到重复啥的说一声，反正都有邮件通知

依我看，有没有协管照样活，除了遇到重复要等别的协管过来

对其他人有意见说出来是没问题的；但是即使你有理那么冲的语气也不自觉会使人家站在你的对立面。

好好的社区被搞得乌烟瘴气……PCL Community 里头都在讨论这事……就加个 tag 删个 tag，开一下关一下改个 boolean 都能闹出那么大的事……

**Reply by iPodtouch6557** [reply]
人多力量大 vs 人越多脑子越少

**Comment by wuliaodexiaoluo** [comment]
我的建议是，如果没有其他事情这件事就此翻篇比较好，继续讨论事态可能会超出社区的控制，乃至造成无法预估的后果（（（

**Comment by MoYuan-CN** [comment]
如果有什么大问题龙猫希望联系我了解/核实/讨论的话，请联系我的正式对外邮箱
- Jim.Lin.4549@outlook.com

**Reply by MoYuan-CN** [reply]
### 重新读完了正文和每一楼之后的统一回复

> \> allMagicNB 于 1 楼内
> 在我问他 Owner 权限什么时候没的时，他又在做什么？在群里闲的聊天都不愿意回我，最后……

回顾了一下那件事情…… 给一个总结吧：
我当时做出操作的决定是出于保护社区（避免因为那件事情给 PCL Community 带来损失）
而产生的。想法是不给已经不是 Collaborator 的成员发 Owner 权限
当时的 PCL Community 还是处于初期，在社区里的人也都是 Collaborator，
所以综合来看我选择了降级为 Member 进行观察

**Reply by MoYuan-CN** [reply]
> \> LTCatt 于 7 楼内
> hmm…… #2863、#3039、#4336 要说 “同一个问题，他连续三次都能查错”，说法上可能有点过火了，……

好的收到了，之后会更加注意
也非常抱歉带来打扰

---

另外就是对每一条回复都用 Emoji 表达了我的看法
👍 表示赞同
👎 表示不赞同
😕 表示将会有长回复
🚀 表示接受观点
👀 表示没有特别的看法

**Reply by MoYuan-CN** [reply]
### 也同时在这里借一楼说点事情

自从 2024 年 5 月 中旬开始，我逐渐恢复了我的正常生活，所以各位会显著的发现我的活跃时间忽然集中在了周末
这个变化意味着我有限的时间和精力需要集中在我的课业上，再加上我脑海中计划了一堆（实际上：19）个项目想做
这导致了我的时间变得非常有限，目前已经很难有机会像以前一样频繁活跃了 😢 

对近期一些大的事情，我也只能做到保持关注，而没有办法花费太多精力在保持收到的信息最新和分析与回复上面
我知道这个帖子迟早会有人发出来，来问我有关我的一些行为。所以我也不会选择回避，而是正面去面对遇到的麻烦
如果各位对我的行为有什么见解、改进建议，抑或是任何你想要表达的内容，
欢迎通过本楼第一楼的邮箱地址联系我，我会认真阅读各位的来信 🧐 

接下来我的想法是在能够保证课业不影响、个人/团队项目正常推进的情况下，力所能及地在 PCL 社区帮忙查重，处理提问等
自这段文章发出之后，我将不会直接关掉我查重完之后的帖子，而是等待另一位协作者来确认是否有误再进一步处理
所以…… 也请不要说我有权限为啥不直接关掉啦 😅 

感谢曾经给过我帮助的每一位社区成员、朋友，和~~曾经~~一起维护社区秩序的每一位协作者

**Reply by wuliaodexiaoluo** [reply]
>如果有什么大问题龙猫希望联系我了解/核实/讨论的话，请联系我的正式对外邮箱
>
>    [Jim.Lin.4549@outlook.com](mailto:Jim.Lin.4549@outlook.com)

~原来换了，我说为什么发的邮件你一封都没回我~
